bitmap-ittybitty/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const handler = require(__dirname + '/lib/handler');
+const app = require(__dirname + '/lib/app');
```

Comment:
If this is a constructor function, you should name it with a capitalized first letter.